doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -193,7 +194,15 @@ See :ref:`cmake_options` for more information about defining CMake options.
 Performing prerequisite operations
 **********************************
 
-You must enable the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option to support the Google Fast Pair standard in your project.
+To add the Google Fast Pair subsystem to your project, you must enable the ``SB_CONFIG_BT_FAST_PAIR`` Kconfig option if you use the default |NCS| build system configuration with sysbuild.
+If you do not use sysbuild, you must enable :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option at the main application image level.
+
+.. note::
+   Sysbuild copies the value of the ``SB_CONFIG_BT_FAST_PAIR`` Kconfig option into the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option that is part of the main application image.
+   Your configuration of the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option at the main application image level has no effect as this Kconfig option is directly set by sysbuild.
+
+Fast Pair API integration
```

Comment:
```suggestion
API
```
?

This part does not describe the whole Fast Pair API integration

Or longer names

Performing prerequisite Kconfig operations
Performing prerequisite API operations

Reply:
You are right that naming may be a bit misleading here (e.g. FMDN APIs are not covered here)